biz.aQute.remote/test/biz/aQute/remote/AgentTest.java

```diff
@@ -186,8 +186,13 @@ public void testAgentUpdateBundleFromURL() throws Exception {
 	public void testAgentSupervisorTimeout() throws Exception {
 		TestSupervisor testSupervisor = new TestSupervisor();
 
-		testSupervisor.connect("localhost", 12345, 300);
-		testSupervisor.connect("localhost", 12345, 100);
+		try {
+			testSupervisor.connect("localhost", 12345, 300);
+		} catch (ConnectException e) {}
+
+		try {
+			testSupervisor.connect("localhost", 12345, 100);
```

Comment:
Lets add fail() here as well to assert that connect() now throws ConnectException